cvat/apps/engine/serializers.py

```diff
@@ -148,52 +151,94 @@ def to_representation(self, instance):
             label['svg'] = instance.skeleton.svg
         return label
 
+    def __init__(self, *args, **kwargs):
+        self._local = kwargs.pop('local', False)
+        """
+        Indicates that the operation is called from the dedicated ViewSet
+        and not from the parent entity, i.e. a project or task.
+        """
+
+        super().__init__(*args, **kwargs)
```

Comment:
Please remove parts about the `_local` variable - they are not relevant to this PR.

Reply:
Removed